tests/acceptance.suite.yml

```diff
@@ -0,0 +1,13 @@
+# Codeception Test Suite Configuration
+#
+# Suite for acceptance tests.
+# Perform tests in browser using the WPWebDriver or WPBrowser.
+# Use WPDb to set up your initial database fixture.
+# If you need both WPWebDriver and WPBrowser tests - create a separate suite.
+
+actor: AcceptanceTester
+modules:
+    enabled:
+        - WPDb
+        - WPBrowser
+        - \Helper\Acceptance
```

Comment:
I got a module not found error with this, do you know which package it's provided by?

Reply:
Which module did it complain about ? the `\Helper\Acceptance` ? I'm guessing you removed the file in `_support` ?